cves/CVE-2013-1664.yml

```diff
@@ -1,296 +1,314 @@
+  code: true
+  code_answer: true
+  fix: true
+  fix_answer: true
+discovered:
+  question: |
+    How was this vulnerability discovered?
+
+    Go to the bug report and read the conversation to find out how this was
+    originally found. Answer in longform below in "answer", fill in the date in
+    YYYY-MM-DD, and then determine if the vulnerability was found by a Google
+    employee (you can tell from their email address). If it's clear that the
+    vulenrability was discovered by a contest, fill in the name there.
+
+    The automated, contest, and developer flags can be true, false, or nil.
+
+    If there is no evidence as to how this vulnerability was found, then please explain where you looked.
+  answer: |
+    Was found by a Yahoo! employee when DOS was plaguing Nova keystone servers
+  automated: false
+  contest: false
+  developer: true
+autodiscoverable:
+  instructions: |
+    Is it plausible that a fully automated tool could have discovered
+    this? These are tools that require little knowledge of the domain,
+     e.g. automatic static analysis, compiler warnings, fuzzers.
+
+    Examples for true answers: SQL injection, XSS, buffer overflow
+
+    Examples for false: RFC violations, permissions issues, anything
+    that requires the tool to be "aware" of the project's
+    domain-specific requirements.
+
+    The answer field should be boolean. In answer_note, please explain
+    why you come to that conclusion.
+  answer_note: |
+    This could be analysed by Fuzzers by finding any http timeout errors, or by
+    monitoring the processes running.
+  answer: true
+specification:
+  instructions: |
+    Is there mention of a violation of a specification? For example,
+    an RFC specification, a protocol specification, or a requirements
+    specification.
+
+    Be sure to check all artifacts for this: bug report, security
+    advisory, commit message, etc.
+
+    The answer field should be boolean. In answer_note, please explain
+    why you come to that conclusion.
+  answer_note: |
+    There was a violation in XML standards that are used.
+  answer: true
+subsystem:
+  question: |
+    What subsystems was the mistake in?
+
+    Most systems don't have a formal list of their subsystems, but you can
+    usually infer them from path names, bug report tags, or other key words
+    used. A single source file is not what we mean by a subsystem. In Django,
+    the "Component" field on the bug report is useful. But there may be other
+    subsystems involved.
+
+    Your subsystem name(s) should not have any dots or slashes in them. Only
+    alphanumerics, whitespace, _, - and @.Feel free to add multiple using a YAML
+    array.
+
+    In the answer field, explain where you saw these words.
+    In the name field, a subsystem name (or an array of names)
+
+    e.g. clipboard, model, view, controller, mod_dav, ui, authentication
+  answer: xmlbomb_20130219.patch

```

Comment:
Did you get this answer from the directories the file was located? Inheritance? Imports?

Reply:
I got these from Django bug commit reports